### ISSUE TITLE
Have "survivalTime" as a variable

### ISSUE THREAD

**Issue by Besfr** [ensingm2/SteamMonsterGameScript#169]
It seems that when you play with a lot of ppl using script we could reduce the survivalTime to have a better economy for damage upgrade.

Is it possible to have this value as a variable ?

**Comment by ensingm2** [comment]
Are you talking about the time until death of the player? Current target? Lane? Level? Game?

Most of these are already implemented around the script in various places, just not accessible to the user. It'd be easy to make them accessible though. What would it be used for? Displayed in stats, or used in some calculation somewhere?

> It seems that when you play with a lot of ppl using script we could reduce the survivalTime to have a better economy for damage upgrade.

Not trying to be mean or anything, but I honestly don't understand this sentence. Could you rephrase?

**Comment by vanZeben** [comment]
Im doing this right now, he means the expected health in the auto upgrade script... Lowering it will make you focus more on dmg upgrades, and less on staying alive for x amount of time

**Comment by ensingm2** [comment]
Ah, yeah sounds good.

**Comment by Besfr** [comment]
Sorry for my bad english ensingm2, vanZeben understand well, i saw that the survivalTime variable is used to determine if the autobuyer script buy a heal upgrade or not.

I'm currently on the room #4 on reddit and my life don't move at all. Maybe it's just a side effect, but there is so many heal that i assume we can find a way to reduce these upgrade and focus a little bit more on damage
![snapshot2](https://cloud.githubusercontent.com/assets/152320/8196492/d66a28e6-148c-11e5-9a84-b31f0b7b460a.png)

I'm not sure if we can add a variance in that, depending of how many heal abilities are used or available.
Also, it seems that buying a heal upgrade heal up to full life, can someone else valid that ? and can we use this to improve the heal autobuyer with some sort of healing manager ?

BTW thx for you awesome work on that script!
